docs/src/conf.py

```diff
@@ -53,12 +53,12 @@
 bibtex_reference_style = "author_year"
 
 sphinx_gallery_conf = {
-    "filename_pattern": r"/*\.py",
+    "examples_dirs": "../../examples",
```

Comment:
The config entries were in alphabetical order (except for the filename_pattern somehow). Maybe we can keep it this way to keep the diffs clean